esphome/components/esp32_rmt_led_strip/led_strip.h

```diff
@@ -96,6 +97,7 @@ class ESP32RMTLEDStripLightOutput : public light::AddressableLight {
   uint16_t num_leds_;
   bool is_rgbw_;
   bool is_wrgb_;
+  bool use_dma_;
```

Comment:
For sake of consistency, could we add initial values to all of these member variables?
```suggestion
  bool use_dma_{false};
```